docs/releases/2.2.txt

```diff
@@ -189,6 +189,9 @@ Models
   :meth:`.QuerySet.bulk_create` to ``True`` tells the database to ignore
   failure to insert rows that fail uniqueness constraints or other checks.
 
+* :class:`~django.contrib.postgres.search.SearchQuery` now produces a more
+  accurate string representation.
```

Comment:
I'm a bit confused. (Need more ☕️ I suppose...) As per @auvipy's previous comment/question, if this is a bug fix for 2.1 doesn't this not belong where you had it (in 2.1.txt)? 

(Only if it's a new feature for 2.2 does it go here, right?)

☕️

Reply:
I got a bit confused! I will add the message to `2.1.1.txt` 👍

Reply:
Wait, I'm not sure. This was added back in 2016 and it's always had the current behavior. I'd classify this as a new feature rather than a bug? I've added it to the changelog for `2.1.1.txt` anyway, let me know what you think.

Reply:
😱 ☕️

Reply:
Lets just go with 2.1.1.txt. (It's a bug fix as classified...) 

(Whilst you're there, can you past-tensify the commit message too)

Thanks!

Reply:
Done! 👍